docs/authzed/pricing.md

```diff
@@ -0,0 +1,93 @@
+
+Yes. Any amount less than 1 million relationships or dispatched operations is considered a charge of `$0`.
+
+## What counts as a "dispatched operation"?
+
+A **dispatched operation** is a single *non-cached* operation performed by Authzed to resolve the answer to the API request.
+
+The number of dispatched operations depends heavily on the type of [API] being called, the complexity of the permissions system's [schema] and how many of those operations have been cached from previous [API] calls.
+
+[schema]: /guides/schema
+
+### API Dispatch Calculation
+
+Below are the (rough) calcuations for determining the worse-cast dispatch count for an API call:
+
+| API | Dispatch Count |
+| ---- | -------------- |
+| [CheckPermission] | The number of non-cached subproblems to be computed |
+| [ExpandPermissionTree] | The number of non-cached subproblems to be computed |
+| [LookupResources] | The number of non-cached subproblems to be computed |
+| [WriteRelationships] | The number of preconditions + `1` for the write |
+| [DeleteRelationships] | The number of preconditions + `1` for the delete |
+
+[CheckPermission]: https://buf.build/authzed/api/docs/main/authzed.api.v1#CheckPermission
+[ExpandPermissionTree]: https://buf.build/authzed/api/docs/main/authzed.api.v1#ExpandPermissionTree
+[LookupResources]: https://buf.build/authzed/api/docs/main/authzed.api.v1#LookupResources
+[WriteRelationships]: https://buf.build/authzed/api/docs/main/authzed.api.v1#WriteRelationships
+[DeleteRelationships]: https://buf.build/authzed/api/docs/main/authzed.api.v1#DeleteRelationships
+
+### Example: CheckPermission
+
+Imagine a permissions system with the following schema:
+
+```zed
+definition user {}
+
+definition document {
+    relation reader: user
+    relation writer: user
+    
+    permission view = reader + writer
+}
+```
+
+Issuing a [CheckPermission] call for `view` permission on a `document` will have to check three sub-problems: `view`, `reader` and `writer`.
+
+The dispatched operation count for this API call will **at most** be `3`, with lower counts based whether the user was found early, whether the problems have been cached, or other optimizations automatically performed by Authzed.
+
+### Finding out the dispatched count
+
+To find the dispatched count of an API operation, the [zed] tool can be used with the `--log-level debug` flag:
+
+```sh
+$ zed permission check document:firstdoc view user:someuser --log-level debug
+DBG set log level new level=debug
+DBG extracted response dispatch metadata cached=3 dispatch=0
+```
+
+The number of dispatched operations can be found in `dispatch`, while the number of *cached* operations can be found as `cached`
+
+:::note
+The number of cached vs dispatched operations will change over time based on the usage of the API
+
+Authzed will automatically cache results and attempt to reuse the cache as much as possible
+:::
+
+[zed]: https://github.com/authzed/zed
+
+## Viewing operation counts
```

Comment:
I think this should be the first heading in the section. Knowing where to check for running costs is probably your first priority, then fully understanding the way it is calculated. (Spoken from someone who has accidentally gotten sticker shock after using a usage based product recently)